src/pipeline_jobs.cpp

```diff
@@ -6641,3 +6641,7 @@ bool RelionJob::getCommandsTomoReconPartJob(std::string &outputname, std::vector
 
 }
 
+std::string RelionJob::obtainFullPathForCommand(const std::string &cmd)
+{
+	return "`which "+ cmd +"`";
```

Comment:
This will be the only row that I need to change when I create a new build of Relion (instead of changing 49 rows).